docs/b2b-edition/specs/storefront/storefront/order.yaml

```
@@ -14,1195 +43,983 @@ security:
-                          type: string
-                          minLength: 1
-                          description: Image URL address
-                      required:
-                        - orderId
-                        - imageUrl
+                    type: object
+                    properties: {}
                 required:
                   - code
                   - message
                   - data
-              examples:
-                example-1:
-                  value:
-                    code: 200
-                    message: Success
-                    data:
-                      - orderId: '123'
-                        imageUrl: url
+  /orders/images:
+    get:
+      summary: Get Order Images
       operationId: get-orders-images
-      parameters:
-        - schema:
-            type: string
-          in: query
-          name: 'orderIds[]'
-          description: Unique order ID
       description: |-
-        Get order images.
+        Retrieves product image URLs based on the `orderIds[]` value in the parameters. You can return images for multiple orders by adding a separate `orderIds[]` parameter for each order.
+
         Equivalent Storefront GraphQL API Query: `orderImages`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
       tags:
-        - Order
-      x-internal: true
+        - Orders
+      parameters:
+        - name: 'orderIds[]'
+          description: |-
+            The BigCommerce order ID. This parameter does not accept the B2B Edition order ID returned in certain [Server to Server Orders endpoints](/b2b-edition/apis/rest-management/order).
+
+            This parameter is not an array, but you can filter for multiple order statuses by including additional `orderIds[]` parameters in the query. For example, `?orderIds[]=123&orderIds[]=124`.
+          schema:
+            type: string
+            example: "123"
+          required: true
+          in: query
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/responseSuccess"
+                  - properties:
+                      data:
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            orderId:
+                              $ref: '#/components/schemas/orderId'
+                            imageUrl:
+                              type: string
+                              minLength: 1
+                              description: The image URL for a product in the order.
+                              example: "https://mybcstore.com/images/products/12345/main.jpg"
+                          required:
+                            - orderId
+                            - imageUrl
```

```suggestion
                  - $ref: "#/components/schemas/responseArray"
                  - properties:
                      data:
                        items:
                          properties:
                            orderId:
                              $ref: '#/components/schemas/orderId'
                            imageUrl:
                              type: string
                              minLength: 1
                              description: The image URL for a product in the order.
                              example: "https://mybcstore.com/images/products/12345/main.jpg"
                          required:
                            - orderId
                            - imageUrl
```
See note above for rationale